integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomainModelInPV.java

```diff
@@ -155,72 +193,116 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
     // create secret for admin credentials
     logger.info("Creating secret for admin credentials");
     adminSecretName = "weblogic-credentials";
-    createSecretWithUsernamePassword(adminSecretName, domainNamespace, "weblogic", "welcome1");
+    createSecretWithUsernamePassword(adminSecretName, domainNamespace, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
 
     // create model encryption secret
     logger.info("Creating encryption secret");
     encryptionSecretName = "encryptionsecret";
     createSecretWithUsernamePassword(encryptionSecretName, domainNamespace, "weblogicenc", "weblogicenc");
 
     // create the PV and PVC to store application and model files
-    createPV(pvName, domainUid, "ItMiiDomainModelInPV");
-    createPVC(pvName, pvcName, domainUid, domainNamespace);
+    createPV(pvName, domainUid1, "ItMiiDomainModelInPV");
+    createPVC(pvName, pvcName, domainUid1, domainNamespace);
 
     // build the clusterview application
     Path distDir = buildApplication(Paths.get(APP_DIR, "clusterview"),
         null, null, "dist", domainNamespace);
     clusterViewAppPath = Paths.get(distDir.toString(), "clusterview.war");
     assertTrue(clusterViewAppPath.toFile().exists(), "Application archive is not available");
 
-    V1Pod webLogicPod = setupPVPod(domainNamespace);
+    logger.info("Setting up WebLogic pod to access PV");
+    V1Pod pvPod = setupWebLogicPod(domainNamespace);
 
+    ExecResult exec = null;
     try {
-      logger.info("Creating directory {0} in PV", "/shared/applications");
-      Exec.exec(webLogicPod, null, false, "/bin/sh", "-c", "mkdir /shared/applications");
+      logger.info("Creating directory {0} in PV", modelMountPath + "/applications");
+      exec = Exec.exec(pvPod, null, false, "/bin/sh", "-c", "mkdir -p " + modelMountPath + "/applications");
+      if (exec.stdout() != null) {
+        logger.info("Exec stdout {0}", exec.stdout());
+      }
+      if (exec.stderr() != null) {
+        logger.info("Exec stderr {0}", exec.stderr());
+      }
     } catch (IOException | ApiException | InterruptedException ex) {
       logger.warning(ex.getMessage());
     }
     try {
-      logger.info("Creating directory {0} in PV", "/shared/model");
-      Exec.exec(webLogicPod, null, false, "/bin/sh", "-c", "mkdir /shared/model");
+      logger.info("Creating directory {0} in PV", modelMountPath + "/model");
+      exec = Exec.exec(pvPod, null, false, "/bin/sh", "-c", "mkdir -p " + modelMountPath + "/model");
```

Comment:
same as above

Reply:
Moved the exec command to CommonTestUtils and checking for non zero exit value, non null/empty stderr to assert the exec is successful.